index.ts

```diff
@@ -195,6 +214,35 @@ const react = async (reaction:MessageReaction, user:User|PartialUser) => {
 
 client.on('message', message);
 client.on('messageReactionAdd', react);
+if ('vcPing' in config) {
```

Comment:
Is there a way to split this into another file? I think the main file is getting long and unwieldly

Reply:
Yeah definitely, but that seems like something for another PR. I'll open a new issue and work on it there

Reply:
Thanks, as long as it gets worked on, I'll submit my approval.